src/util/mock-servers/service-testers/service-handlers/implementation/read-rows-handler.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import {SameCallHandler} from './same-call-handler';
+import {MockService} from '../../../mock-service';
+import {Mutation} from '../../../../../mutation';
+
+function rowResponse(rowKey: {}) {
```

Comment:
Is there going to be a different "implementation" file for different test cases, e.g, ensuring that streaming errors are gracefully recovered from?

If so, I would be tempted to name the implementation file after the specific behaviour it's a test for. Then you'll have a bunch of implementation files for a bunch of different regression tests.